config/config_baseline.yml

```diff
@@ -0,0 +1,27 @@
+data:
+  path_to_data: data
+  train_file: train_data_toy.parquet # this is for the train_test_split.py output
+  test_file: dev_data_toy.parquet # this is for the train_test_split.py and eval_f1.py output
```

Comment:
I suggest specifying the exact same params reproducing the baseline:

- test_file: dev_data.parquet
- num_train_epochs: 20
- etc.